.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+.classpath
```

Comment:
Very good point on putting .classpath in .gitignore. I'll do it for the other connectors too. 👍